src/api/app/assets/stylesheets/webui/application.scss

```diff
@@ -62,6 +62,7 @@
 
 @import 'responsive_ux/responsive_ux';
 @import 'notifications_redesign/notifications';
+@import 'user_profile_redesign/user_profile';
```

Comment:
I would avoid add an "empty" css file for the future. I would add it when we really need it

Reply:
Alright, I'm taking out the corresponding commit, I'll add it in the next PR if needed.